package.json

```diff
@@ -61,7 +61,7 @@
     "@pagopa/io-functions-commons": "^20.6.6",
     "@pagopa/ts-commons": "^9.6.0",
     "abort-controller": "^3.0.0",
-    "applicationinsights": "^1.7.4",
+    "applicationinsights": "^1.8.10",
```

Comment:
Why this change?

Reply:
Because there are other dependencies that carry out `applicationinsights` ' `1.8.10` version, so I've updated it, since version `1.7.4` caused a mismatch of telemetry Client type. It was `NodeClient | DefaultClient` instead of simply `NodeClient`